.github/workflows/ci.yml

```
@@ -0,0 +1,42 @@
+            - name: WordPress Plugin Check
```

actionlint (1.7.4)

workflow is sequence node but mapping node is expected

(syntax-check)